src/altinn-app-frontend/src/features/form/components/MessageBanner.tsx

```diff
@@ -7,14 +7,14 @@ import classNames from 'classnames';
 
 const useStyles = makeStyles({
```

Comment:
Maybe the Grid should be a header container?

Reply:
Please elaborate on that?

Reply:
I think I put this comment on the wrong location.
Sorry.

Reply:
I think I meant that the component in this file could be a header instead of a div.
Only based it on the name of the file containing banner, which is often in a header.